bootstrap/less/forms.less

```diff
@@ -36,6 +36,12 @@ label {
   font-weight: bold;
 }
 
+small {
+  display: inline-block;
+  color: lighten(@legend-color, 50%);
+  font-style: italic;
+  font-weight: lighter;
+}
```

Comment:
These styles won't work for all of the themes that are bundled with this theme. In addition, I don't think we should be attaching styles to all `<small>` tags in case people want to use them elsewhere. See next comment.